include/ft_mlx.h

```diff
@@ -6,22 +6,42 @@
 /*   By: msousa <mlrcbsousa@gmail.com>              +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2021/11/03 15:08:07 by msousa            #+#    #+#             */
-/*   Updated: 2021/11/09 22:30:03 by msousa           ###   ########.fr       */
+/*   Updated: 2022/05/01 18:19:38 by msousa           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef FT_MLX_H
 # define FT_MLX_H
 
-# include <mlx.h>
-# include "keys.h"
-
 # define BUTTON_PRESS 4
 # define DESTROY_NOTIFY 17
 
 # define SCROLL_UP 4
 # define SCROLL_DOWN 5
 
+/* Keys */
+# ifdef OS_MAC
+#  define KEY_UP 	126
+#  define KEY_DOWN 	125
+#  define KEY_LEFT 	123
+#  define KEY_RIGHT 124
+#  define KEY_W 	13
+#  define KEY_A 	0
+#  define KEY_S 	1
+#  define KEY_D 	2
+#  define KEY_SPACE 49
+#  define KEY_ENTER 36
+#  define KEY_ESC 	53
+# else
+#  define KEY_UP	65362
+#  define KEY_DOWN	65364
+#  define KEY_LEFT	65361
+#  define KEY_RIGHT	65363
+#  define KEY_ESC	65307
+# endif
```

Comment:
for Windows:

KEY_W 119
KEY_A 97
KEY_S 115
KEY_D 100

KEY_SPACE 32
KEY_ENTER 65293